### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by haacked**
Adding a readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You needz a readme yo.
```
Comment (FakeHaacked) [inline README.md:1]: `needz`? I don't speak `l33t`.

Comment (haacked) [inline README.md:1]: test

**Comment by FakeHaacked** [comment]
another test

**Comment by FakeHaacked** [comment]
yet another test

**Comment by haacked** [comment]
Ha

**Comment by haacked** [comment]
ha ha

**Comment by FakeHaacked** [comment]
aoeuoaeuoeu

**Comment by FakeHaacked** [comment]
aoeuaoeuoaeuoaeuoeu

File: README.md (hunk 1 above)
Comment (FakeHaacked) [inline README.md:1]: aoeuoaeu

**Pull request merged**